clj-rn.conf.edn

```diff
@@ -75,6 +75,8 @@
                    "react-native-desktop-menu"
                    "react-native-desktop-config"
                    "react-native-desktop-notification"
+                   "react-native-link-preview"
+                   "react-native-scalable-image"
```

Comment:
is it possible to use RN image ? what the point to use this library?

Reply:
RN doesn't allow you to display image from internet and set only `width` property. It require you to specify also `height`, which is unknown for remote image. And if you specify arbitrary `height`, then image will have free white spaces around it if not all height used.
 This library solves the problem.

Reply:
i believe there should be a way how to do that with RN image, we're trying not to use external libraries because of security issues

Reply:
yeah, if you take a look at the source, it's super simple to implement in cljs https://github.com/ihor/react-native-scalable-image/blob/master/index.js

Reply:
@flexsurfer, I think that decision on including/not including 3rd-party modules should be made after every library review.

`react-native-scalable-image` as you pointed is fairy simple and probably has no security issues. Also its version will be set in stone with `yarn.lock`. What is the point in reimplementing it with cljs? 

If we have issues with current implementation and want to change it - thats ok. But replicating the behavior in other language just to get rid of record in` package.json` - that sounds like a waste of time for me. Wdyt? Maybe I miss some knowledge about how exactly 3rd party modules introduce issues?

Reply:
let's discuss this with other team members in status maybe, because from my pov it's better and faster to implement it in clojure than include a separate library